src/directory/directory.mjs

```diff
@@ -820,6 +820,66 @@ export const directory = {
           }
         ]
       },
+      logging: {
+        title: 'Logging',
+        items: [
+          {
+            title: 'Set up Logging',
+            route: '/lib/logging/setup-logging',
+            filters: ['ios', 'android']
+          },
+          {
+            title: 'Send Logs',
+            route: '/lib/logging/sending-logs',
+            filters: ['ios', 'android']
+          },
+          {
+            title: 'Change Log Levels',
+            route: '/lib/logging/change-log-levels',
+            filters: ['ios', 'android']
+          },
+          {
+            title: 'Flush Logs',
+            route: '/lib/logging/flush-logs',
+            filters: ['ios', 'android']
+          },
+          {
+            title: 'Enable/Disable Logging',
+            route: '/lib/logging/enable-disable',
+            filters: ['ios', 'android']
+          },
+          {
+            title: 'Configure User Allow List',
+            route: '/lib/logging/configure-user',
+            filters: ['ios', 'android']
+          },
+          {
+            title: 'Remotely Change Log Levels',
+            route: '/lib/logging/remote-configuration',
+            filters: ['ios', 'android']
+          },
+          {
+            title: 'Access Logs',
```

Comment:
can we move the access logs which will now be "Viewing logs" to right before the remote configuration item?

Reply:
Moved this page to before remote configuration.